lib/presentation/screen/calendar/custom_calendar_card.dart

```diff
@@ -19,15 +19,12 @@ class CustomCalendarCard extends StatelessWidget {
       children: [
         GestureDetector(
           onTap: () {
-            Navigator.push(
-              context,
-              MaterialPageRoute(
-                builder: (context) => DaysOfMonth(
-                  year: year,
-                  month: monthIndex,
-                ),
-              ),
-            );
+            ///Todo: it will give error because it takes arguments
+            context.goNamed('MonthScreen', params: {
+              'year': year.toString(),
+              'month': monthIndex.toString(),
+              'isHomeScreen': false.toString(),
+            });
```

Comment:
do not create maps. create class instead for Args

Reply:
OKay